src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -974,13 +975,21 @@ grpc_error_handle Chttp2ServerAddPort(Server* server, const char* addr,
   // Using lambda to avoid use of goto.
   grpc_error_handle error = [&]() {
     grpc_error_handle error;
-    if (absl::ConsumePrefix(&parsed_addr_unprefixed, kUnixUriPrefix) ||
-        absl::ConsumePrefix(&parsed_addr_unprefixed, kUnixAbstractUriPrefix) ||
-        absl::ConsumePrefix(&parsed_addr_unprefixed, kVSockUriPrefix)) {
-      absl::StatusOr<EventEngine::ResolvedAddress> result =
-          grpc_event_engine::experimental::URIToResolvedAddress(parsed_addr);
-      GRPC_RETURN_IF_ERROR(result.status());
-      results->push_back(*result);
+    // TODO(ladynana, yijiem): this code does not handle address URIs correctly:
```

Comment:
I would add here that part of this fix needs to be adding API documentation that `grpc_server_add_http2_port` expects a valid URI.

Reply:
done.